src/electionguard_cli/setup_election/setup_election_command.py

```diff
@@ -52,12 +62,10 @@ def SetupElectionCommand(
     """
 
     setup_inputs = SetupInputRetrievalStep().get_inputs(
-        guardian_count,
-        quorum,
-        manifest,
+        guardian_count, quorum, manifest, url
     )
     joint_key = KeyCeremonyStep().run_key_ceremony(setup_inputs.guardians)
-    build_election_results = ElectionBuilderStep().build_election_with_key(
+    build_election_results = SetupElectionBuilderStep().build_election_for_setup(
```

Comment:
Following up on other comment, could just be `build_election_for_setup(  )`.

Reply:
It could, but I'm not sure this is the PR to rearchitect the CLI, I'm just looking to meet a need.  I will add a ticket to do that refactor if you feel strongly about it.